crypto/core_namemap.c

```diff
@@ -137,6 +137,10 @@ int ossl_namemap_doall_names(const OSSL_NAMEMAP *namemap, int number,
     cbdata.number = number;
     cbdata.found = 0;
 
+    if (namemap == NULL) {
+        return 0;
+    }
```

Comment:
OpenSSL style is to not include `{}` for a single line if.

Reply:
Ok, fixed.